platformio.ini

```diff
@@ -1391,14 +1418,22 @@ extra_scripts        = ${common.extra_scripts}
 #
 [env:mks_robin_pro2]
 platform             = ${common_stm32.platform}
+platform_packages    = framework-arduinoststm32@https://github.com/rhapsodyv/Arduino_Core_STM32/archive/usb-host-msc.zip
 extends              = common_stm32
-build_flags          = ${common_stm32.build_flags} -DHAL_HCD_MODULE_ENABLED -DUSBHOST
+build_flags          = ${common_stm32.build_flags}
+  -DHAL_PCD_MODULE_ENABLED
+  -DUSBHOST 
+  -DHAL_HCD_MODULE_ENABLED
+  -DUSBH_IRQ_PRIO=3
+  -DUSBH_IRQ_SUBPRIO=4
```

Comment:
These could probably be moved into `common_stm32.build_flags`. They won't always be needed, but they probably do no harm.

Reply:
I think we should wait my stm32buino be merged before doing any change on "default" envs. This names could change.

Reply:
Have you heard any timeline for this? Is merging these now a good idea, or should we keep this in draft until we have an updated stm32duino?

Reply:
They will add my PR on the 2.0.0 release. There's no date yet. That is the reason I think we should merge it, using separated envs.